tools/sdk-testgen/packages/autorest.testmodeler/src/common/constant.ts

```diff
@@ -9,6 +9,8 @@ export enum Config {
     disabledExamples = 'testmodeler.mock.disabled-examples',
     sendExampleId = 'testmodeler.mock.send-example-id',
     verifyResponse = 'testmodeler.mock.verify-response',
+    scenarioCodeModelRestCallOnly = 'testmodeler.scenario.codemodel-restcall-only',
+    useExampleModel = 'testmodeler.use-example-model',
 }
```

Comment:
Could you please add doc for these two flags to the README.md?

Reply:
doc added